apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -241,22 +402,54 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                 if (!get(groupedReceipts, 'length')) {
                     throw new GQLError(errors.MISSING_REQUIRED_GROUPED_RECEIPTS, context)
                 }
-                if (groupedReceipts.some(group => !get(group, ['receipts', 'length']))) {
-                    throw new GQLError(errors.MISSING_REQUIRED_RECEIPTS_IN_GROUPED_RECEIPTS, context)
+
+                const billingReceiptsKey = groupedReceipts.some(group => get(group, ['billingReceipts']))
+                    ? 'billingReceipts'
+                    : groupedReceipts.some(group => get(group, ['receipts']))
+                        ? 'receipts'
+                        : 'billingReceipts'
+                const serviceConsumerKey = groupedReceipts.some(group => get(group, ['serviceConsumer']))
+                    ? 'serviceConsumer'
+                    : groupedReceipts.some(group => get(group, ['consumerId']))
+                        ? 'consumerId'
+                        : 'serviceConsumer'
```

Comment:
you need to mark this code as remove for future

Reply:
done